Automattic/jetpack
# VideoPress : Fix semicolon displayed on the rendered page

#### Changes proposed in this Pull Request:

This PR fixes a typo rendering a semicolon in Gutenberg when adding a new Video block

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a Jetpack site with VideoPress enabled
* Create a new post or edit an existing one
* Add a /video block
* ✅ No ";" should appear after the block in the DOM (and on the rendered page)
